debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbSnapshotChangeEventSource.java

```diff
@@ -377,7 +378,7 @@ private void createDataEventsForReplicaSet(ChangeEventSourceContext sourceContex
                                     snapshotReceiver,
                                     replicaSet,
                                     id,
-                                    primaryClient);
+                                    nodeClient);
```

Comment:
Maybe just "client" like elsewhere and reserve "node" for PreferredNode / PrimaryNode?